packages/fast-components-react-base/src/listbox-item/listbox-item.tsx

```diff
@@ -24,25 +25,60 @@ class ListboxItem extends Foundation<
         disabled: false,
     };
 
+    /**
+     * Gets the id to use for the item
+     * Authors who's items all have unique values can choose to
+     * not provide an itemId and the values of the items will be used as id's,
+     * otherwise use itemId.  All listbox items of a particular listbox should use the same
+     * approach with respect to using values or itemId or results could be unexpected.
+     * The "id" prop is being depracated and should not be used.
```

Comment:
spelling, deprecated

Reply:
"deprecated" is clearly my spelling bee weakness.